tests/outputs/preslist/presentations.aux

```diff
@@ -0,0 +1,6 @@
+\relax 
```

Comment:
why do we have a  latex aux file in the repo?  Isn't this a file created during latex compilation?